client/src/Components/Navbar/style.js

```diff
@@ -0,0 +1,25 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
```

Comment:
pass the theme as a parameter to your makeStyle function, then get color from the theme, palette
``` 
const useStyles = makeStyles((theme) => ({
```